src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,54 @@
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../utils/searchHelper';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sex = searchParams.get('sex') || '';
+
+  const onQueryChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchParams(
+      getSearchWith(searchParams, { query: e.currentTarget.value || null }),
+    );
+  };
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        <Link
+          className={classNames({ 'is-active': sex === '' })}
+          to={{
+            search: getSearchWith(searchParams, {
+              sex: null,
+            }),
+          }}
+        >
+          All
+        </Link>
+        <Link
+          className={classNames({ 'is-active': sex === 'm' })}
+          to={{
+            search: getSearchWith(searchParams, {
+              sex: 'm',
+            }),
+          }}
+        >
+          Male
+        </Link>
+        <Link
+          className={classNames({ 'is-active': sex === 'f' })}
```

Comment:
You can move these links to separate component, as the only difference between them is text and letter (either no letter, 'm', or 'f').